### PR TITLE
ADD: Add citation file

### LINKED ISSUE #25: FR: Create DOI through zenodo
We should mint a DOI of this work:

https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content

### PR DESCRIPTION
**Pull request by mgrover1**
Add a citation file the repo

cc @MathewBiddle @kellijohnson-NOAA - can you check your information to make sure it is correct?

Closes #25

### CODE REVIEW

File: CITATION.cff
Comment (kellijohnson-NOAA) [inline CITATION.cff]: I just changed jobs on Monday so I am now
NOAA Office of Science and Technology

Comment (mgrover1) [reply CITATION.cff]: Congrats on the new gig! I can update accordingly! Thank you for working with us on this project!!

**Comment by mgrover1** [comment]
Awaiting approval from @MathewBiddle here before we merge!

**Review by MathewBiddle — APPROVED** [review]

**Comment by MathewBiddle** [comment]
Looks great! Thanks for adding it.

**Pull request merged**
